.github/workflows/prerelease.yml

```diff
@@ -147,14 +140,6 @@ jobs:
         shell: bash
         run: |
           build/windows/upload_msi.sh ${INTEGRATION} ${{ matrix.goarch }} ${TAG}
-      - name: Notify failure via Slack
```

Comment:
why removing this? i think this is working https://newrelic.slack.com/archives/C01516RVD4H/p1684393634080039

Reply:
answered on slack thanks